Formula/libb64.rb

```diff
@@ -0,0 +1,20 @@
+class Libb64 < Formula
+  desc "Base64 encoding/decoding library"
+  homepage "https://libb64.sourceforge.io/"
+  url "https://downloads.sourceforge.net/project/libb64/libb64/libb64/libb64-1.2.src.zip"
+  sha256 "343d8d61c5cbe3d3407394f16a5390c06f8ff907bd8d614c16546310b689bfd3"
+
+  def install
+    system "make"
+    bin.mkpath
+    bin.install "base64/base64"
+    include.mkpath
+    include.install "include/b64"
+    lib.mkpath
+    lib.install "src/libb64.a"
+  end
+
+  test do
+    system "#{bin}/base64", "-e", "/dev/null", "/dev/null"
```

Comment:
Could we create something more useful here involving assertions? Like maybe create a file, feed that in, and see if the output is as expected?

Reply:
For my other tools, I need only libb64. For that purpose starting base64 seems to be a good enough test.  However, feel free to add a more elaborate test, if you like.

Reply:
Right, gotcha. That's fine.

I'll have a look later this week before merging. Thanks for your first pull request @schloegl!

Reply:
> For my other tools, I need only libb64. For that purpose starting base64 seems to be a good enough test. However, feel free to add a more elaborate test, if you like.

This definitely needs a better test. Homebrew maintainers typically have their hands pretty full, so we'll probably need your help @schloegl to get this going.

If you only need the library, you could write a small C program in the tetst (or similar) to ensure you can link against the library and/or install a simple test into the `pkg_share` directory during install and then compile & link in the test.

Reply:
> This definitely needs a better test. Homebrew maintainers typically have their hands pretty full, so we'll probably need your help @schloegl to get this going.

Yes.




> For that purpose starting base64 seems to be a good enough test.

This starts it, does nothing and verifies an exit code. It'd be preferable to use `assert_match` and `shell_output` (and shouldn't take long to do).

Reply:
> If you only need the library, you could write a small C program in the tetst (or similar) to ensure you can link against the library and/or install a simple test into the `pkg_share` directory during install and then compile & link in the test.


I really need only libb64.a.  Can you point me to some example  (e.g. in some other formula, or in the homebrew documentation) how such a test should look like ?

Reply:
We can have an additional test for the library if desired (see `libxml2.rb` for a library test) but this formula will need to have a test for the binary too and need it to be installed, sorry.